frontend/src/api/endpoints.ts

```diff
@@ -4,14 +4,13 @@ export const AUTH = {
   LOGOUT: "/auth/logout/",
   TOKEN_REFRESH: "/auth/token/refresh/",
   PROFILE: "/auth/profile/",
-  // Future endpoints 
-//   PASSWORD_RESET: "/auth/password/reset/",
-//   VERIFY_EMAIL: "/auth/verify-email/",
-//   OAUTH_GOOGLE: "/auth/oauth/google/",
-//   OAUTH_MICROSOFT: "/auth/oauth/microsoft/",
-//   OAUTH_CALLBACK: "/auth/oauth/callback/",
+  SIDEBAR: "/auth/sidebar/",
```

Comment:
```suggestion
  COURSES: "/courses/",
```
Should be like this to fetch courses but we can also fix this later when courses are merged in the backend. I think this should be good tho